<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpriteEditor</class>
 <widget class="QMainWindow" name="SpriteEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SpriteEditor</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="SpriteCanvas" name="canvasWidget" native="true">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>60</y>
      <width>325</width>
      <height>325</height>
     </rect>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>410</y>
      <width>621</width>
      <height>141</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>619</width>
       <height>139</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="SpritePreview" name="previewWidget" native="true">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>10</y>
      <width>130</width>
      <height>130</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="previewSpeedController">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>150</y>
      <width>130</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>280</y>
      <width>70</width>
      <height>280</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="copyGroupButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Copy Group</string>
    </property>
   </widget>
   <widget class="QLabel" name="cpInstructionsLabel">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Pixels copied! Click to choose an anchor point and then click where you wish to paste.</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="addFrameButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Frame</string>
    </property>
   </widget>
   <widget class="QPushButton" name="copyFrameButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Copy Frame</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteFrameButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Frame</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="eraseButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Erase</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="groupSelectButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Pixels</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="bucketFillButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Bucket Fill</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="penToolButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Pen Tool</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="colorButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="ColorLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Segoe UI&quot;;
text-decoration: underline;</string>
    </property>
    <property name="text">
     <string>Color Picker</string>
    </property>
   </widget>
   <widget class="QLabel" name="explanationLabel">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>No pixels to copy. Please select pixels.</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SpriteCanvas</class>
   <extends>QWidget</extends>
   <header>spritecanvas.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SpritePreview</class>
   <extends>QWidget</extends>
   <header location="global">spritepreview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
